### PR TITLE
refactor: split test_flux_analysis.py to separate modules.

### PR DESCRIPTION
**Pull request by synchon**
Addresses https://github.com/opencobra/cobrapy/issues/742. Currently, refactors `test_flux_analysis.py`.

### CODE REVIEW

File: cobra/test/test_flux_analysis/conftest.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+@pytest.fixture(scope="session",
+                params=[s for s in ["cplex", "gurobi"]
+                        if s in sutil.solvers])
+def qp_solvers(request):
+    """Return the available QP solvers."""
+    return request.param
+
+
+@pytest.fixture(scope="session")
```
Comment (Midnighter) [inline cobra/test/test_flux_analysis/conftest.py]: Please move this fixture into the `test_room` module and change the scope to `"module"`.

File: cobra/test/test_flux_analysis/conftest.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    b_2.reaction = "E ->"
+    b_3.reaction = "byp ->"
+
+    test_model.objective = 'b2'
+
+    return test_model
+
+
+@pytest.fixture(scope="session")
+def room_solution():
```
Comment (Midnighter) [inline cobra/test/test_flux_analysis/conftest.py]: Same as above.

File: cobra/test/test_flux_analysis/conftest.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                            'v6': 0.0})
+    sol = Solution(objective_value=5.000, status='optimal',
+                   fluxes=fluxes,
+                   reduced_costs=reduced_costs,
+                   shadow_prices=shadow_prices)
+    return sol
+
+
+@pytest.fixture(scope="session")
+def geometric_fba_model():
```
Comment (Midnighter) [inline cobra/test/test_flux_analysis/conftest.py]: Same but move it to the respective module, please.

File: cobra/test/test_flux_analysis/conftest.py (hunk 1 above)
Comment (synchon) [reply cobra/test/test_flux_analysis/conftest.py]: Sure!

Comment (synchon) [reply cobra/test/test_flux_analysis/conftest.py]: I totally forgot to mention that `room_model` and `room_solution` fixtures are needed in `test_deletion.py` too. Will it be okay to keep these fixtures as they are?

File: cobra/test/test_flux_analysis/conftest.py (hunk 3 above)
Comment (synchon) [reply cobra/test/test_flux_analysis/conftest.py]: I can definitely move this though.

File: cobra/test/test_flux_analysis/conftest.py (hunk 1 above)
Comment (Midnighter) [reply cobra/test/test_flux_analysis/conftest.py]: Okay, then keep them in the `conftest.py` but still change the scope to `"module"`, please.

Comment (synchon) [reply cobra/test/test_flux_analysis/conftest.py]: Done!

**Pull request merged**